app/controllers/incident_reports_controller.rb

```diff
@@ -15,7 +15,7 @@ def update
       else redirect_to @incident,
                   notice: 'Incident report was successfully saved.'
       end
-    else render 'incidents/edit'
+    else render 'edit'
```

Comment:
Why does `incidents` not matter anymore? Rails stuff?

Reply:
Previously `incidents/edit` contained the edit forms for the incident, the driver account of incident, the supervisor account of incident, and the supervisor report. This PR breaks each of them off into their own edit page.

For this reason, we don't want to always send people back to the top-level page anymore.

Reply:
OHHH

Reply:
Yo sé.

Reply:
Usted sabe? O usted ve? 😛